.github/workflows/add-preview-domain.yml

```diff
@@ -8,10 +8,54 @@ permissions:
   contents: read
 
 jobs:
+  comment-jira-ticket:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Comment with the jira ticket
+        uses: actions/github-script@v7.0.1
+        with:
+          script: |
+            const branch = `${{ github.head_ref }}`
+            const match = branch.match(/\b(MI|AS)-\d+\b/);
+            const jiraTicket = match ? match[0] : null;
+
+            if (!jiraTicket) {
+              console.log('No Jira ticket found')
+              return
+            }
+
+            const knownString = '### Jira ticket'
+            const pullRequest = await github.rest.pulls.get({
+              pull_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            const hasAlreadyCommented = pullRequest.data.body ? pullRequest.data.body.includes(knownString) : false
+
+            if (hasAlreadyCommented) {
+              console.log('Already commented once')
```

Comment:
What if task description changes

Reply:
Next time the PR syncs, it'll re-run and add it if it got removed.